ras_backstage/__init__.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from flask import Flask
+from flask_cors import CORS
+from flask_restplus import Api, Namespace
+
+from ras_backstage.logger_config import logger_initial_config
+
+app = Flask(__name__)
+
+app_config = 'config.{}'.format(os.environ.get('APP_SETTINGS', 'Config'))
+app.config.from_object(app_config)
+
+app.url_map.strict_slashes = False
+
+logger_initial_config(service_name='ras-frontstage-api', log_level=app.config['LOGGING_LEVEL'])
```

Comment:
Why is the service name frontstage-api and not ras-backstage?

Reply:
Because I keep copy/pasting this 🤦‍♂️